draft-ietf-rtcweb-jsep.xml

```diff
@@ -3131,9 +3132,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           </list></t>
 
           <t>Once all the session-level lines have been parsed,
-          processing continues with the lines in media sections.</t>
+          processing continues with the lines in m= sections.</t>
         </section>
-        <section title="Media Section Parsing"
+        <section title="M= Section Parsing"
```

Comment:
Not super excited about this change. Since these terms are isomorphic I think it would be fine to use 'm= section' when talking about a specific section and 'media section' when being abstract.